### ISSUE TITLE
Add ability to toggle background checkmark and cross

### ISSUE THREAD

**Issue by th3fallen** [aaronshaf/react-toggle#56]
Is there a way to diable the checkmark and x when you toggle this? would be super nice

Or is hiding them via css the recommended way?

**Comment by ahstro** [comment]
~~There's a PR (#34) open for this; I just recently told them to fix some things so I can merge it, but if they don't respond within a week(ish), I'll just go ahead and merge it myself. In the meantime, hiding them via CSS is super simple, just set `display: none` for `.react-toggle-track-check` and `.react-toggle-track-x`.~~

Edit: I thought you were asking _how_ to hide them via CSS. I just realized that I don't even know if CSS might be the recommended way to do this. @aaronshaf, what do you think?

**Comment by th3fallen** [comment]
@ahstro thanks for the info and your first response was correct i just wanted to hide them whichever way was recommended. i've done css for now but i'll keep an eye out for #34 and when it merges go that route.

Thanks.

**Comment by ahstro** [comment]
Let's keep this open until the issue is really resolved :)

**Comment by ahstro** [comment]
Hmm, I'm wondering whether the best alternative here is a simple boolean saying whether icons should be shown (#34) or allowing custom checked and unchecked values - which then can be set to `null` - as in #30? What do people think? @th3fallen @tow8ie @bnadeau @aaronshaf 

I think I'm leaning more towards custom values, possibly passed in as

``` js
icons: PropTypes.shape({
  checked: PropTypes.node,
  unchecked: PropTypes.node,
})
```

with default props

``` js
icons: {
  checked: <Check />,
  unchecked: <X />,
}
```

This way, we would allow setting custom content, but also easily disable icons with `<Toggle icons={false} />` if we use `icons ? icons.checked : null`, or some equivalent, to render.

**Comment by th3fallen** [comment]
@ahstro i think the latter of the two would be best for flexibility. so my vote goes for #30's approach

**Comment by ahstro** [comment]
Okay, I think we're going with what I wrote above. If @tow8ie or @bnadeau wants to write this, speak up, otherwise I'm going to start working on this as soon as I'm done writing some tests. I'll be quite busy this weekend, but I'm going to try to squeeze something in; I want to release 2.2.0 as soon as possible.

Edit: If anyone else wants to give it a shot, that's great too (PRs are always appreciated), I just `@`ed tow8ie and bnadeau because they have similar PRs open already.

**Comment by ahstro** [comment]
Implemented in #70
